### DISCUSSION TITLE
Help me with training food101 mini model.

### DISCUSSION THREAD

**Discussion by shisirkha** [mrdbourke/pytorch-deep-learning#761] [General]
As I saw that how much I try. or even how much I give it the balance it gives me poor results like 0.23022% for example
and can you guys please tell what can i add to my layers to improve my model to the next level i tried adding dropout layers and it gave me a slight fraction of boost but still it's too low. can you guys tell me how can i add more and more unique layers to my model so I can improve it.

thanks for your help

**Comment by shisirkha** [comment]
Guys good news, i got significant improvement by resizing the image transform to a higher resolution then 64 like 300
loss:
![image](https://github.com/mrdbourke/pytorch-deep-learning/assets/111275085/7fc56bfb-d533-4337-9a78-d74cd0d42eca)

accuracy:
![image](https://github.com/mrdbourke/pytorch-deep-learning/assets/111275085/a30d031b-a335-402d-888b-9ba74919bf3c)

**Comment by shisirkha** [comment]
Guys i found out that if we use adamax optimizer it will increase our predictions.

**Reply by shisirkha** [reply]
It works for me.

**Comment by shisirkha** [comment]
I think i found out that
